tests/ha/test_server_pod_failure.py

```diff
@@ -626,7 +603,7 @@ def test_degraded_deletes_safe_server_pod_shutdown(self):
         LOGGER.info("Completed READs on remaining %s .", remain_bkt)
         LOGGER.info("Deleting remaining %s buckets.", remain_bkt)
         args = {'test_prefix': self.test_prefix, 'test_dir_path': self.test_dir_path,
-                'skipput': True, 'skipget': True, 'bkts_to_del': remain_bkt, 'output': del_output}
```

Comment:
so we don't want to delete remain buckets as logger says?

Reply:
Updated.